server/views/components/sort-search/sort-search.config.js

```diff
@@ -0,0 +1,27 @@
+export const context = {
+  model: {
+    inputs: [
+      {
+        label: 'Best match',
+        type: 'radio',
+        id: 'sort-best-match',
+        name: 'sort',
+        value: ''
+      },
+      {
+        label: 'Newest',
+        type: 'radio',
+        id: 'sort-newest',
+        name: 'sort',
+        value: 'DESC'
+      },
+      {
+        label: 'Oldest',
+        type: 'radio',
+        id: 'sort-oldest',
+        name: 'sort',
+        value: 'ASC'
+      }
+    ]
+  }
+};
```

Comment:
Why not build these directly into the component?
I can't really see the benefit of this being config?